app-config.yaml

```diff
@@ -155,6 +155,7 @@ catalog:
       target: https://github.com/spotify/backstage/blob/master/packages/catalog-model/examples/acme-corp.yaml
 
 scaffolder:
+  # workingDirectory: /tmp # Use this to configure a working direcotry for the scaffolder, defaults to the OS temp-dir
```

Comment:
Given https://github.com/spotify/backstage/issues/2998#issuecomment-716557710, should we move this to sit under `backend` instead of `scaffolder`?

Reply:
@Rugvip is that a something that is already used or would we be introducing it now?

Reply:
Yep, unless theres a reason to configure two different places. @spotify/techdocs-core thoughts?

Reply:
Sounds right to move this config to `backend`.

Reply:
How could I forget that backend already existed..

I will move it to the backend part then.